de.dashup.application/src/main/webapp/components/dashup-radio-button.js

```diff
@@ -0,0 +1,19 @@
+import {LitElement, html, css} from "https://unpkg.com/lit-element@2.1.0/lit-element.js?module";
+import {DashupComponent} from "./dashup-component.js";
+export class DashupRadioButton extends DashupComponent{
```

Comment:
Why does the radio button not have a render function? Is this not implemented yet or not necessary?

Reply:
Rendering takes place in the Dashup Radio Button Group. These custom elements will be the children of the group, defining how the group should be rendered.